js/main.js

```diff
@@ -16,18 +16,19 @@ var HttpClient = function() { // Thanks http://stackoverflow.com/a/22076667/1709
 
 function getLunchInfo(lunchMenu) { // Bypassing CORS using JSONP
   try {
-    var lunchSpecial = (lunchMenu.days[today.getDay() - 1].menu_items[1].food.name);
-
-    document.getElementById('lunch-body').innerHTML = lunchSpecial;
+    document.getElementById('lunch-body').innerHTML = (lunchMenu.days[today.getDay() - 1].menu_items[1].food.name);
   } catch (error){
     document.getElementById('lunch-body').innerHTML = "No Lunch Served";
   }
 }
 
 function clock(isHalfDay){
   // Thanks to http://stackoverflow.com/a/36524883/1709894 and https://www.w3schools.com/howto/howto_js_countdown.asp
+  const start = new Date();
+  start.setHours(7, 45, 0, 0);
+  const startTime = start.getTime();
   var countDownDate = new Date();
-  var time =+ countDownDate;
+  var time = +countDownDate;
```

Comment:
Yea. This is actually correct. We were right that += is correct, but +date actually is a typecast. This is saying turn countDownDate into milliseconds.